src/visualizer/sortingvisualizer/component/multipleblocks/StackOfBoxes.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import HighlightedOval from '../block/HighlightedOval';
+
+/**
+ * Stack of boxes of an item in the bucket for radix sort.
+ *
+ * @component
+ * @category MultipleBlocks
+ * @param {Object} individualStack A single stack in the buckets.
```

Comment:
is indivudalStack supposed to be any[] instead? just like in StackOfAnimatedBoxes

Reply:
Nope, this is an Object.